packages/react-atlas-core/src/Checkbox/Checkbox.js

```diff
@@ -32,7 +32,7 @@ class Checkbox extends React.PureComponent {
 
       this.setState({ checked: !this.state.checked }, function() {
         this._validationHandler(this.props.errorCallback);
-
+        debugger;
```

Comment:
please remove debug code.

Reply:
@ericsewalson weird, this is not the latest state of the code, on my local i don't have this